elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -694,6 +712,8 @@ public String getNameFromAlias(Object entity, String alias) {
             Initializer<T> initializer = getEntityBinding(entity.getClass()).getInitializer();
             if (initializer != null) {
                 initializer.initialize(entity);
+            } else if (injector != null) {
```

Comment:
@aklish what do we expect if both are defined?

Reply:
The initializer overrides the injector.  There is only one way to initialize something - either a general injector or a class specific initializer.